core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -68,6 +71,10 @@
     private static final AtomicIntegerFieldUpdater<DefaultRequestLog> flagsUpdater =
             AtomicIntegerFieldUpdater.newUpdater(DefaultRequestLog.class, "flags");
 
+    private static final AtomicReferenceFieldUpdater<DefaultRequestLog, HttpHeaders>
+            responseTrailersUpdater = AtomicReferenceFieldUpdater.newUpdater(
+            DefaultRequestLog.class, HttpHeaders.class, "responseTrailers");
```

Comment:
I guess we do not need CAS because trailers are always expected to get updated from an I/O thread.